src/js/rx/containers/Active.jsx

```diff
@@ -162,11 +162,6 @@ class Active extends React.Component {
         <div>
           <p className="rx-tab-explainer">Your active VA prescriptions</p>
           {this.renderViewSwitch()}
-          {this.state.view === 'list' || <SortMenu
-              onChange={this.handleSort}
-              onClick={this.handleSort}
-              options={sortOptions}
-              selected={currentSort}/>}
```

Comment:
Moved the sort menu to the `PrescriptionList` component since it's specifically used there.